app/views/works/homepage.html.erb

```diff
@@ -0,0 +1,43 @@
+<div>
+<h1>Media Spotlight: <%= Work.spotlight.title %> by <%= Work.spotlight.creator %></h1>
+  <%= Work.spotlight.votes.count %> votes - <%= Work.spotlight.description %>
+
+</div>
+
+<section class="top-ten-list-container">
+
+  <section class="top-ten-list">
```

Comment:
Note that these 3 blocks of code for each category are nearly the same. Consider how you could DRY this up with a partial view or loop.